org.omg.sysml.interactive/src/org/omg/sysml/interactive/SysMLInteractiveUtil.java

```diff
@@ -45,14 +47,17 @@ private static String formatRelationship(Relationship relationship) {
 	}
 	
 	private static void formatElement(StringBuilder buffer, String indentation, Element element, String relationshipTag) {
-		String shortName = element.getShortName();
-		String name = nameOf(element);
-		buffer.append(indentation + 
-				relationshipTag + 
-				element.eClass().getName() + 
-				(shortName == null? "": " <" + shortName + ">") +
-				(name == null? "": " " + name) + 
-				" (" + element.getElementId() + ")\n");
+		buffer.append(indentation + relationshipTag + element.eClass().getName());		
+		if (EvaluationUtil.isMetaclassFeature(element)) {
+			formatElement(buffer, " ", ((MetadataFeature)element).getAnnotatedElement().get(0), "");
```

Comment:
It is not particularly important. It just includes the represented element in the display of a metaclass feature, e.g., "MetadataFeature PartDefinition P", rather than just "MetadataFeature".